### PR TITLE
Add aliases

### PR DESCRIPTION
**Pull request by SamirTacos**
Added aliases from Alias Megathread

### CODE REVIEW

File: data/aliases.ts [hunk 1]
```diff
@@ -85,6 +85,8 @@ export const Aliases: {[alias: string]: string} = {
 	mayhem: "[Gen 9] Random Battle Mayhem",
 	omotm: "[Gen 9] Frantic Fusions",
 	lcotm: "[Gen 7] Pure Hackmons",
+	cgt: "[Gen 9] Computer-Generated Teams",
+	compgen: "[Gen 9] Computer-Generated Teams",
```
Comment (KrisXV) [inline data/aliases.ts:89]: i would prefer these to be near the rands formats

File: data/aliases.ts [hunk 2]
```diff
@@ -109,6 +112,8 @@ export const Aliases: {[alias: string]: string} = {
 	megasnow: "Abomasnow-Mega",
 	megashark: "Sharpedo-Mega",
 	megasaur: "Venusaur-Mega",
+	mewtwox, "Mewtwo-Mega-X",
+	mewtwoy, "Mewtwo-Mega-Y",
```
Comment (MathyFurret) [inline data/aliases.ts:116]:
```suggestion
	mewtwox: "Mewtwo-Mega-X",
	mewtwoy: "Mewtwo-Mega-Y",
```

File: data/aliases.ts [hunk 3]
```diff
@@ -94,6 +96,7 @@ export const Aliases: {[alias: string]: string} = {
 	megacharizard: "Charizard-Mega-Y",
 	megacharizardx: "Charizard-Mega-X",
 	megacharizardy: "Charizard-Mega-Y",
+	megacruci: "Crucibelle-Mega",
```
Comment (KrisXV) [inline data/aliases.ts:99]: maybe a new section for cap mons mightbe worth it? i dont wanna conflate cap and real pokemon

File: data/aliases.ts (hunk 2 above)
Comment (KrisXV) [inline data/aliases.ts:116]: this isnt proper syntax

File: data/aliases.ts [hunk 4]
```diff
@@ -293,7 +301,21 @@ export const Aliases: {[alias: string]: string} = {
 	taurospaldea: "Tauros-Paldea-Combat",
 	taurospaldeafire: "Tauros-Paldea-Blaze",
 	taurospaldeawater: "Tauros-Paldea-Aqua",
-
+	venomicone: "Venomicon-Epilogue",
```
Comment (KrisXV) [inline data/aliases.ts:304]: again, cap section

File: data/aliases.ts [hunk 5]
```diff
@@ -293,7 +301,21 @@ export const Aliases: {[alias: string]: string} = {
...
+	ogerpongrass: "Ogerpon",
+	ogerponteal: "Ogerpon",
+	ogerpontealtera: "Ogerpon-Teal-Tera",
+	ogerpongrasstera: "Ogerpon-Teal-Tera",
+	ogerponrock: "Ogerpon-Cornerstone",
+	ogerponrocktera: "Ogerpon-Cornerstone-Tera",
+	ogerponfire: "Ogerpon-Hearthflame",
+	ogerponfiretera: "Ogerpon-Hearthflame-Tera",
+	ogerponwater: "Ogerpon-Wellspring",
+	ogerponwater: "Ogerpon-Wellspring-Tera",
```
Comment (KrisXV) [inline data/aliases.ts:317]: this is the same entry twice

File: data/aliases.ts [hunk 6]
```diff
@@ -293,7 +301,21 @@ export const Aliases: {[alias: string]: string} = {
...
 	taurospaldeafire: "Tauros-Paldea-Blaze",
 	taurospaldeawater: "Tauros-Paldea-Aqua",
-
+	venomicone: "Venomicon-Epilogue",
+	glowking: "Slowking-Galar",
+	glowbro: "Slowbro-Galar",
+	enamorust: "Enamorus-Therian",
+	ogerpongrass: "Ogerpon",
+	ogerponteal: "Ogerpon",
+	ogerpontealtera: "Ogerpon-Teal-Tera",
```
Comment (KrisXV) [inline data/aliases.ts:310]: this is the same thing

File: data/aliases.ts [hunk 7]
```diff
@@ -346,6 +368,12 @@ export const Aliases: {[alias: string]: string} = {
 	ufof: "Pokestar UFO",
 	ufoflying: "Pokestar UFO",
 	vivillonmeadow: "Vivillon",
+	venomiconp: "Venomicon",
+	venomiconprologue: "Venomicon",
```
Comment (KrisXV) [inline data/aliases.ts:372]: cap thing

File: data/aliases.ts [hunk 8]
```diff
@@ -346,6 +368,12 @@ export const Aliases: {[alias: string]: string} = {
...
+	enamorusincarnate: "Enamorus",
+	enamorusincarnation: "Enamorus",
+	ogerponteal: "Ogerpon",
+	ogerpongrass: "Ogerpon",
```
Comment (KrisXV) [inline data/aliases.ts:376]: you already put these in earlier

File: data/aliases.ts [hunk 9]
```diff
@@ -570,8 +598,10 @@ export const Aliases: {[alias: string]: string} = {
 	av: "Assault Vest",
 	balloon: "Air Balloon",
 	band: "Choice Band",
+	booster: "Booster Energy",
 	boots: "Heavy-Duty Boots",
 	cb: "Choice Band",
+	cc: "Covert Cloak",
```
Comment (KrisXV) [inline data/aliases.ts:604]: this is already an alias for close combat

File: data/aliases.ts [hunk 10]
```diff
@@ -693,64 +724,86 @@ export const Aliases: {[alias: string]: string} = {
...
 	camel: "Camerupt",
 	cathy: "Trevenant",
 	chandy: "Chandelure",
 	chomp: "Garchomp",
 	clanger: "Kommo-o",
 	clef: "Clefable",
+	clod: "Clodsire",
 	coba: "Cobalion",
 	cofag: "Cofagrigus",
+	colo: "Colossoil",
```
Comment (KrisXV) [inline data/aliases.ts:755]: cap section

File: data/aliases.ts [hunk 11]
```diff
@@ -693,64 +724,86 @@ export const Aliases: {[alias: string]: string} = {
...
 	conk: "Conkeldurr",
+	corvi: "Corviknight",
 	cress: "Cresselia",
+	croak: "Toxicroak",
+	cruci: "Crucibelle",
```
Comment (KrisXV) [inline data/aliases.ts:760]: cap section

File: data/aliases.ts [hunk 12]
```diff
@@ -693,64 +724,86 @@ export const Aliases: {[alias: string]: string} = {
...
 	darm: "Darmanitan",
 	dnite: "Dragonite",
 	dogars: "Koffing",
 	don: "Groudon",
+	doom: "Houndoom",
 	drill: "Excadrill",
 	driller: "Excadrill",
 	dug: "Dugtrio",
 	duggy: "Dugtrio",
+	ebook: "Venomicon-Epilogue",
```
Comment (KrisXV) [inline data/aliases.ts:773]: cap section

**Review by KrisXV — CHANGES_REQUESTED** [review]
I dont feel the need to comment on everything, but there are a lot of standards in this file as notated in the comments that got ignored with a lot of basic syntax errors that make it obvious that this wasnt tested locally, so im going to put this back into a draft for now

File: data/aliases.ts (hunk 9 above)
Comment (MathyFurret) [inline data/aliases.ts:604]:
```suggestion
```
CC is already used for Close Combat

File: data/aliases.ts [hunk 13]
```diff
@@ -693,64 +724,86 @@ export const Aliases: {[alias: string]: string} = {
...
 	chandy: "Chandelure",
 	chomp: "Garchomp",
 	clanger: "Kommo-o",
 	clef: "Clefable",
+	clod: "Clodsire",
 	coba: "Cobalion",
 	cofag: "Cofagrigus",
+	colo: "Colossoil",
 	conk: "Conkeldurr",
+	corvi: "Corviknight",
```
Comment (MathyFurret) [inline data/aliases.ts:757]:
```suggestion
	corv: "Corviknight",
```
This is more common

File: data/aliases.ts [hunk 14]
```diff
@@ -895,11 +976,13 @@ export const Aliases: {[alias: string]: string} = {
 	sr: "Stealth Rock",
 	ssa: "Shell Side Arm",
 	sub: "Substitute",
+	tarrows, "Thousand Arrows",
```
Comment (MathyFurret) [inline data/aliases.ts:979]:
```suggestion
	tarrows: "Thousand Arrows",
```

File: data/aliases.ts [hunk 15]
```diff
@@ -805,28 +869,39 @@ export const Aliases: {[alias: string]: string} = {
 	skarm: "Skarmory",
 	smogon: "Koffing",
 	smogonbird: "Talonflame",
+	smoko: "Smokomodo",
+	snael, "Snaelstrom",
```
Comment (MathyFurret) [inline data/aliases.ts:873]:
```suggestion
	snael: "Snaelstrom",
```

**Review by MathyFurret — CHANGES_REQUESTED** [review]
quick pass

File: data/aliases.ts (hunk 9 above)
Comment (MathyFurret) [reply data/aliases.ts:604]: Could add "cloak" if you really want an alias for this

File: data/aliases.ts (hunk 6 above)
Comment (MathyFurret) [reply data/aliases.ts:310]:
```suggestion
```
completely redundant alias

File: data/aliases.ts [hunk 16]
```diff
@@ -805,28 +866,39 @@ export const Aliases: {[alias: string]: string} = {
...
+	thorns: "Iron Thorns",
 	thundy: "Thundurus",
 	toed: "Politoed",
+	tomb: "Spiritomb",
 	torn: "Tornadus",
 	tran: "Heatran",
+	treads: "Iron Treads",
 	ttar: "Tyranitar",
+	tusk: "Great Tusk",
+	valiant: "Iron Valiant",
```
Comment (smudgerox) [inline data/aliases.ts:890]: "ival" and "val" are both very common shorthands for this mon

File: data/aliases.ts [hunk 17]
```diff
@@ -848,17 +920,23 @@ export const Aliases: {[alias: string]: string} = {
 	bb: "Brave Bird",
 	bd: "Belly Drum",
 	bde: "Baby-Doll Eyes",
+	blades: "Precipice Blades",
```
Comment (smudgerox) [inline data/aliases.ts:923]: pblades is another good alias for this

**Comment by Zarel** [comment]
Superseded by #10378

**Pull request closed without merging**